onnxruntime/test/featurizers_ops/truncated_svdtransformer_test.cc

```diff
@@ -46,9 +46,25 @@ void TruncatedSVDTransformerTestRowMajStandard() {
   OpTester test("TruncatedSVDTransformer", 1, onnxruntime::kMSFeaturizersDomain);
   test.AddInput<uint8_t>("State", {dim}, stream);
   test.AddInput<T>("Input", {3, 3}, {-1, -1, 0, 0, -2, -1, -3, 0, -2});
-  test.AddOutput<T>("Output", {3, 3}, {1.009107f, 0.626315f, 0.767745f, 0.965105f, 1.995869f, -0.291682f, 3.529165f, -0.724887f, -0.139759f});
-  test.SetOutputAbsErr("Output", 0.00016f);
-  test.Run();
+  std::vector<T> output = {-1.009107f, 0.626315f, -0.767745f, -0.965105f, 1.995869f, 0.291682f, -3.529165f, -0.724887f, 0.139759f};
+  test.AddOutput<T>("Output", {3, 3},
+                    output);
+
+  Matrix verify_matrix(output.data(), 3, 3);
+
+  // Custom verification function
+  OpTester::CustomOutputVerifierFn ver_fn = [&verify_matrix](const std::vector<OrtValue>& fetches, const std::string&) {
```

Comment:
Explaining why we need this would be good.

Reply:
see: [https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=125902](https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=125902)

It fails on Linux x64 too. 

1: /onnxruntime_src/onnxruntime/test/providers/provider_test_utils.cc:158: Failure
1: The difference between expected[i] and output[i] is 0.00021243095397949219, which exceeds expected_data.absolute_error_.value(), where
1: expected[i] evaluates to -0.72488701343536377,
1: output[i] evaluates to -0.72509944438934326, and
1: expected_data.absolute_error_.value() evaluates to 0.00015999999595806003.


So you should relax the tolerance regardless the sizeof(size_t). 

---
In reply to: [400587483](https://github.com/microsoft/onnxruntime/pull/3341#discussion_r400587483) [](ancestors = 400587483)

Reply:
Discussed offline. The reference above is to an old build where standard verification function was used. The new verification employs variance across whole marix and tolerance need only be relaxed on x86.

---
In reply to: [401172884](https://github.com/microsoft/onnxruntime/pull/3341#discussion_r401172884) [](ancestors = 401172884,400587483)